locale/en/docs/meta/topics/dependencies.md

```diff
@@ -26,7 +26,7 @@ There are several dependencies that Node.js relies on to work the way it does.
 The V8 library provides Node.js with a JavaScript engine, which Node.js
 controls via the V8 C++ API. V8 is maintained by Google, for use in Chrome.
 
-* [Documentation](https://v8docs.nodesource.com/)
+* [Documentation](https://v8.dev/docs)
```

Comment:
Just an observation but the old URL and new URL are pointing at different things. The old URL was a link to V8 API documentation, which is useful to people embedding V8 (e.g. Node.js core developers) or writing addons (although `N-API` or using `nan` would be preferable to directly using the V8 API's to protect addons from breaking when the V8 API changes). 

The new URL doesn't contain any of this API documentation, although there is an issue requesting it (https://github.com/v8/v8.dev/issues/234).

Reply:
@richardlau: This was fixed ;)

Reply:
> 
> 
> @richardlau: This was fixed ;)

I see no difference from when I maded the observation -- the contents of the new link are different to the contents of the old link.